src/ServiceControl.Audit.AcceptanceTests/Setup/DirectoryDeleter.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace ServiceControl.Audit.AcceptanceTests.Setup
+{
+    using System.Diagnostics;
+    using System.IO;
+    using System.Linq;
+    using System.Security.AccessControl;
+    using System.Security.Principal;
+
+    static class DirectoryDeleter
+    {
+        public static void Delete(string path)
```

Comment:
Could you add a comment here describing why this is needed vs using APIs? That could be helpful for the next person that works on these tests :)

Reply:
done.